Cargo.toml

```diff
@@ -359,6 +359,8 @@ selinux = { workspace=true, optional = true }
 textwrap = { workspace=true }
 zip = { workspace=true, optional = true }
 
+help_parser = { package="help_parser", path="src/help_parser" }
```

Comment:
`package=` should not be necessary right if the name already matches right?

Reply:
Yes, you're right, it's not needed. I will remove it.